perun-base/src/main/resources/perun-roles.yml

```diff
@@ -6477,6 +6477,7 @@ perun_policies:
 
   getAllVos_policy:
     policy_roles:
+      - SELF:
```

Comment:
there's no filter applied to the result of the method and the method itself returns all vos, so if possible, do not add the SELF role here, or apply filter if needed

Reply:
SELF is gone.